src/app/index.tsx

```diff
@@ -9,13 +9,14 @@ import { Visualization } from './visualization';
 import { Advance } from './advance';
 
 export const App: React.FunctionComponent = () => {
-  const { progress } = useSelector(gameStateSelector);
+  const gameState = useSelector(gameStateSelector);
+  const visualizationArgs = { gameState };
 
   return (
     <div>
-      <Visualization />
+      <Visualization {...visualizationArgs} />
```

Comment:
I think this might be a little more readable if it was just `<Visualization gameState={gameState} />`, what do you think?

Reply:
I agree, I figured this format required less code change to update `<Visualization />` prop arguments in case they changed frequently during initial development but I doubt there will be many.